src/app/src/index.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { render } from 'react-dom';
+import ReactDOM from 'react-dom';
```

Comment:
I think this change may be extraneous -- it looks like we're still using both `render` by itself and `ReactDOM.render` below.

Reply:
Nevermind -- I misread what was happening below. Conceivable we could call the new, wrapped `render` something more specific: `renderWithHotReloading` or something -- to avoid name collisions.

Reply:
Good idea. Will do.